### ISSUE TITLE
Invalid code (PhySL) generation for multi-value return statements.

### ISSUE THREAD

**Issue by rtohid** [STEllAR-GROUP/phylanx#595]
If a function returns a tuple or, equivalently, multiple values, only the last value is returned in PhySL.
```py
from phylanx import Phylanx
import numpy as np

@Phylanx(debug=True)
def foo(x, y):
    return (x, y)

x = 1
y = 2
print(foo(x, y))
```
corresponding PhySL which returns `2`:
```
define(foo, x, y, x, y)
```

**Comment by hkaiser** [comment]
Should'nt this be:

**Comment by hkaiser** [comment]
@rtohid any news on this?

**Comment by hkaiser** [comment]
This was fixed by merging #595
